### ISSUE TITLE
wrong-file-owner-uid-or-gid

### ISSUE THREAD

**Issue by 7ranceaddic7** [phoronix-test-suite/phoronix-test-suite#54]
I downloaded the package from your site.  I have all .deb packages set to install through the Ubuntu Software Center.  I get this:

Lintian check results for /home/scott/phoronix-test-suite_5.4.1_all.deb:
E: phoronix-test-suite: wrong-file-owner-uid-or-gid etc/ 1000/1000 
...
E: phoronix-test-suite: wrong-file-owner-uid-or-gid usr/share/phoronix-test-suite/pts-core/commands/download_test_files.php 1000/1000

Basically, every file's permissions is improperly set for a deb package install through the USC.  A quick run through lintian identifies incorrect permissions, again, for all files.   I know this doesn't mean it can't be installed.  Just thought you should know for those who don't realize what is happening and would opt  not to install.
